front_end/webserver.py

```diff
@@ -1074,7 +1074,7 @@ def post(self, course, assignment, exercise):
                                 if total_size > 10 * 1024 * 1024:
                                     result = f"Error: Your total file size is too large ({total_size} bytes)."
 
-                            if exercise_details["hold_output_constant"]:
+                            if exercise_basics['exists'] and exercise_details["hold_output_constant"]:
```

Comment:
This fixes a bug where the following lines of code were reached when creating a new exercise (when hold_output_constant isn't even visible yet)